src/screens/ModelPage/ModelPage_Phase3.js

```diff
@@ -0,0 +1,241 @@
+
+  let countyOption = null;
+
+  if (countyId) {
+    countyOption = _.find(
+      US_STATE_DATASET.state_county_map_dataset[stateId].county_dataset,
+      ['county_url_name', countyId],
+    );
+  }
+  const [selectedCounty, setSelectedCounty] = useState(countyOption);
+
+  useMemo(() => {
+    setSelectedCounty(countyOption);
+  }, [countyOption]);
+  const history = useHistory();
+
+  const projections = useProjections(stateId, selectedCounty);
+  const stateSummary = useStateSummary(stateId);
+
+  const stateName = STATES[stateId];
+  const intervention = STATE_TO_INTERVENTION[stateId];
+
+  const goTo = route => {
+    history.push(route);
+  };
+
+  // Projections haven't loaded yet
+  // If a new county has just been selected, we may not have projections
+  // for the new county loaded yet
+  if (!projections || projections.county !== selectedCounty) {
+    return <LoadingScreen></LoadingScreen>;
+  }
+
+  let actionTitle;
+  let actionDescription;
+  if (intervention === INTERVENTIONS.SHELTER_IN_PLACE) {
+    actionTitle = `${stateName}: Keep staying at home to protect against the COVID-19 outbreak.`;
+    actionDescription = `Avoiding hospital overload depends heavily on your cooperation.`;
+  } else {
+    actionTitle = `${stateName}: Urge your public officials to act now against the COVID-19 outbreak!`;
+    actionDescription = `To prevent hospital overload, our projections indicate a Stay at home order must be implemented soon.`;
+  }
+  let metaTags = (
+    <AppMetaTags
+      canonicalUrl={`/us/${stateId.toLowerCase()}`}
+      pageTitle={`${stateName} Forecast`}
+      pageDescription={actionTitle}
+      shareTitle={actionTitle}
+      shareDescription={actionDescription}
+    />
+  );
+
+  const renderHeader = () => {
+    return (
+      <SearchHeaderWrapper>
+        <SearchHeader
+          setMapOption={setMapOption}
+          mobileMenuOpen={mobileMenuOpen}
+          setMobileMenuOpen={setMobileMenuOpen}
+        />
+      </SearchHeaderWrapper>
+    );
+  };
+
+  const chart_modules = [
+    {
+      title: '[DEV] Projections',
+    },
+    {
+      title: '[DEV] Case Growth Rate',
+    },
+    {
+      title: '[DEV] Testing',
+    },
+  ];
+  const renderMainContent = () => {
+    return (
+      <MainContentWrapper mobileMenuOpen={mobileMenuOpen}>
+        <MainContentInner>
+          <LocationPageHeader projections={projections} />
+          <MainContentInnerBody>
+            {chart_modules.map((chart_module, index) => {
+              return (
+                <>
+                  {projections && (
+                    <ChartModule
+                      title={chart_module.title}
+                      series={['test']}
+                    ></ChartModule>
+                  )}
+                </>
+              );
+            })}
+          </MainContentInnerBody>
+        </MainContentInner>
+      </MainContentWrapper>
+    );
+  };
+
+  const renderMapContent = () => {
```

Comment:
Consider exporting the render functions from `ModelPage` that are not changed and using them here to avoid duplication and to make sure that they stay consistent (until we change them).

Reply:
ah i think these are pretty nested within the react component itself so it would be hard to remove? but I agree with the principle!

Reply:
okay i was confused but this is now done!